frontend/src/Frontend/UI/Dialogs/AddVanityAccount.hs

```diff
@@ -22,15 +22,15 @@ import Frontend.UI.Widgets.Helpers (dialogSectionHeading)
 import Frontend.Network
 import Frontend.Wallet
 
-uiAddAccountButton
+uiWatchAccountButton
   :: forall t m key mConf
      . ( MonadWidget t m, Monoid mConf
        , HasModalCfg mConf (ModalImpl m key t) t
        )
   => ModalIde m key t
   -> m mConf
-uiAddAccountButton m = do
-  eOpenAddAccount <- uiButton headerBtnCfg  (text "+ Add Account")
+uiWatchAccountButton m = do
+  eOpenAddAccount <- uiButton headerBtnCfg  (text "+ Watch Account")
```

Comment:
Seems like [this line](https://github.com/kadena-io/chainweaver/blob/446544ba607922e6dbed38bb8f8c301c1f49a0f7/frontend/src/Frontend/UI/Wallet.hs#L183) needs to change as well. Without that, the No Accounts Found message refers to an "+ Add Account button".

![image](https://user-images.githubusercontent.com/1258139/179986125-de8975dc-0132-4170-9d5f-889eedfbf6d6.png)